### ISSUE TITLE
Stereo Tool doesn't work on ARM anymore

### ISSUE THREAD

**Issue by BeyondChrispy** [AzuraCast/AzuraCast#6606]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

#919e9b5 (2023-09-05 19:22)

### What happened?

I'm no longer able to use Stereo Tool on an ARM Docker build of AzuraCast.  I've managed to reproduce this on both Oracle Cloud and Amazon AWS using the same setup (not imported from backup, created from scratch with the same settings).  This works fine on x64.

I'm getting the following error in the Liquidsoap log over and over:
`Error 14: Uncaught runtime error:
type: invalid,
message: "Invalid stereotool library",
stack: at audio.liq, line 232, char 6-73,
at audio.liq, line 232, char 6-73,
at /var/azuracast/stations/landmark/config/liquidsoap.liq, line 269 char 8 - line 274 char 1`

I've installed the current 'https://download.thimeo.com/Stereo_Tool_Generic_plugin.zip' which is recommended in the AzuraCast interface.

I recall that in previous versions, I would install the 'https://www.stereotool.com/download/stereo_tool_pi2_64' file. This no longer is accepted by AzuraCast.

The issue only occurs when a Stereo Tool configuration file is present - I've tried many different ones, including standard presets and others of mine that work fine elsewhere.  When I remove the configuration file and reload, the station works (with no processing).

This issue has been present in most builds from the past 9 months or so - I've just left the affected instances on a working build but it's now getting to a point where I would like to use new features.

### Relevant log output

```shell
Error 14: Uncaught runtime error:
type: invalid,
message: "Invalid stereotool library",
stack: at audio.liq, line 232, char 6-73,
at audio.liq, line 232, char 6-73,
at /var/azuracast/stations/landmark/config/liquidsoap.liq, line 269 char 8 - line 274 char 1
```

**Comment by BusterNeece** [comment]
@BeyondChrispy We may not be clear enough in our documentation here, but if you're on ARM machine, the version of the plugin you should be pulling is the "Raspberry Pi" version on Thimeo's site. That contains the ARM/ARM64 binaries that we need and should fix that issue.

**Comment by BeyondChrispy** [comment]
@BusterNeece Yes, I always used to install the Raspberry Pi version, but now, Azuracast doesn't accept that file - only the 300MB 'Stereo_Tool_Generic_plugin.zip'.

If I try to install the 'stereo_tool_pi2_64' or 'stereo_tool_pi2' plugin, Azuracast gives the following error:
![image](https://github.com/AzuraCast/AzuraCast/assets/59484113/f1b504f3-6d3e-416c-96a7-c14d4c68fd34)

Installing the 'Stereo_Tool_Generic_plugin.zip', succeeds but then breaks Liquidsoap.

On the Stereo Tool site, the download links for 'Raspberry Pi Thimeo-ST plugin' and 'x86/64 Linux Thimeo-ST plugin' lead to the same 300MB zip file.

![image](https://github.com/AzuraCast/AzuraCast/assets/59484113/37ef438f-71ff-4da7-84d3-496d9d7c30ee)

![image](https://github.com/AzuraCast/AzuraCast/assets/59484113/1cd7fe22-3c0f-4b2d-9c8f-17cf0a5d637c)

**Comment by BeyondChrispy** [comment]
Pleased to say this looks to be resolved in the latest rolling release #e1300ed (2023-09-26 15:16)!

Thanks!
